Add Digests for CiVo and Myks

linusfr opened the pull request:
Sorry, missed this during my initial PR adding myks.

Also: can you tell me how I kick of the auto-versioning? The nightly run didn't fetch all the versions for some reasons.
And when I am running `hermit -d manifest auto-version --update-digests civo.hcl` or `hermit -d manifest auto-version --update-digests myks.hcl` locally they also don't update the files and the output is `debug: Auto-versioning /<path-to-repo>/hermit-packages/<package-name>.hcl`. 

I can't find anything in the documentation to help me here. Help would be greatly appreciated :) 
Thanks

alecthomas commented:
`strip = 0` is the default, so this will be a noop.

Auto-versioning won't backfill versions. You can use `hermit manifest update-digests mkys.hcl` to just update digests.

linusfr commented:
Oh well, then I must have made some other mistake the last time around. I removed the strip line.

Concerning digests: thanks for the add-digests command. That works. I added digests for myks and civo on here. Is that okay or shall I open a different PR?
Older versions aren't required, as long as future versions will be added. Good to know :)

Thanks for the swift responses and the help 👍🏼

alecthomas commented:
Fine in this PR, thanks!
